### ISSUE TITLE
Custom and likelihood

### ISSUE THREAD

**Issue by thoree** [magnusdv/pedprobr#4]
``` r
library(pedmut)
library(pedprobr)
#> Loading required package: pedtools
n = 3  #Works for n = 1, 2
als = 1:n
p = rep(1/n, n)
H2 = nuclearPed(father = "NN", mother = "Mother", child = "Child")
M = matrix(ncol = n, nrow = n, 1/n)
M = mutationMatrix("custom", matrix = M, alleles = als)
mH2 = marker(H2,  Mother = 1, Child = 1, 
             alleles = als, afreq = p, mutmod = M)
likelihood(H2, mH2)
#> Error in names(lumpedFreq) <- colnames(newM): 'names' attribute [2] must be the same length as the vector [1]
```

Created on 2018-09-26 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

**Comment by magnusdv** [comment]
This was a `pedmut` problem. Should be fixed with the commit magnusdv/pedmut@0ed8fb9.

``` r
library(pedmut)

m = matrix(1/3, ncol = 3, nrow = 3)
M = mutationMatrix("custom", alleles = 1:3, matrix = m)

lumpedMatrix(M, lump = 2:3)
#>              1      lump
#> 1    0.3333333 0.6666667
#> lump 0.3333333 0.6666667
#> 
#> Model: custom 
#> Rate: NA 
#> 
#> Lumpable: Always
```

**Comment by thoree** [comment]
Fine - thanks!
Thore

Den tor. 27. sep. 2018 kl. 10:11 skrev Magnus Dehli Vigeland <
notifications@github.com>:
